data/sql/updates/pending_db_world/rev_1583667245426891200.sql

```diff
@@ -402,8 +401,8 @@ UPDATE `creature_template` SET `mindmg` = 327, `maxdmg` = 434, `DamageModifier`
 UPDATE `creature_template` SET `mindmg` = 331, `maxdmg` = 381, `DamageModifier` = 1 WHERE `entry` = 9499;
 -- Phalanx
 UPDATE `creature_template` SET `mindmg` = 658, `maxdmg` = 873, `DamageModifier` = 1 WHERE `entry` = 9502;
--- Hurley Blackbreath (Not listed in Beastiary) Src: https://wotlk.evowow.com/?npc=9537 / https://db.rising-gods.de/?npc=9537
-UPDATE `creature_template` SET `mindmg` = 705, `maxdmg` = 935, `DamageModifier` = 1 WHERE `entry` = 9537; 
+-- Hurley Blackbreath (Not listed in Beastiary) Src: https://classicdb.ch/?npc=9537
+UPDATE `creature_template` SET `mindmg` = 526, `maxdmg` = 626 WHERE `entry` = 9537;
```

Comment:
downscaled, is it normal? looks okay to me

btw the new damage look more logical than maxdmg 935